tests/utils.py

```diff
@@ -198,3 +200,16 @@ def create_iamc_query_test_data(test_mp):
     run2.meta = {"run": 2, "test": "string", "bool": False}
 
     return [r1, r2, r3], units
+
+
+def create_indexsets_for_run(
+    platform: Platform, run_id: int, amount: int = 2, offset: int = 0
+) -> tuple[IndexSet, ...]:
+    """Create `amount` indexsets called `Indexset n` for `run` (n in (offset,
+    offset+amount])."""
```

Comment:
To keep it in one line...
```suggestion
    """Create `amount` indexsets called `Indexset n` for a `run`."""
```

Reply:
That's what I had originally and then I thought I should clarify what `amount` and `offset` are, too. But maybe we can consider them self-explanatory.

Reply:
I think there's beauty in one-line docstrings, and the function is really short enough that it's easy enough to understand.

Reply:
Agreed, changed it in all place where this function is introduced :)